packages/block-library/src/navigation/edit.js

```diff
@@ -64,15 +59,13 @@ function Navigation( {
 				{
 					backgroundColor: true,
 					textColor: true,
-					fontSize: fontSize.size,
```

Comment:
With this new setup, how can we pass the font size value to the color panel contrast checker? I noticed that `ColorEdit ` and `ColorPanel` components don't explicitly deal with font size, and it's not being passed to the contrast checker in e.g. the Paragraph and the Heading block color panels.

Reply:
Cool, is this mentioned in any existing issue or shall I create a new one?

Reply:
I am not aware of an issue for it so I guess we should create a new one.

Reply:
Done: #24607